src/Layout/redist/MSBuildImports/Current/Microsoft.Common.CrossTargeting.targets/ImportAfter/Microsoft.TestPlatform.CrossTargeting.targets

```diff
@@ -34,14 +34,24 @@ Copyright (c) .NET Foundation. All rights reserved.
   -->
 
   <Target Name="DispatchToInnerBuildsWithVSTestTarget" Returns="@(InnerOutput)">
+    <PropertyGroup>
+      <!-- 
+        Emulating technique found in Microsoft.Build.Traversal for deciding whether to run tests in parallel.
+        Setting either property will revert to previous behavior of running tests in parallel, but without their
+        different TFMs running in parallel.
+      -->
+      <_TestTfmsInParallel>$([MSBuild]::ValueOrDefault('$(TestTfmsInParallel)', '$(BuildInParallel)'))</_TestTfmsInParallel>
+    </PropertyGroup>
     <ItemGroup>
       <_TargetFramework Include="$(TargetFrameworks)" />
+      <_MSBuildProjectFile Include="$(MSBuildProjectFile)" Properties="TargetFramework=%(_TargetFramework.Identity);VSTestNoBuild=true" />
```

Comment:
this is a pretty generic Item name - maybe rename to something more specific like `_ProjectToTestWithTFM`?

Reply:
done.